spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/security/oauth2/resource/servlet/OAuth2ResourceServerAutoConfigurationTests.java

```diff
@@ -473,6 +475,10 @@ void autoConfigurationShouldConfigureResourceServerUsingJwkSetUriAndIssuerUri()
 				assertThat(context).hasSingleBean(JwtDecoder.class);
 				JwtDecoder jwtDecoder = context.getBean(JwtDecoder.class);
 				assertThat(jwtDecoder).extracting("jwtValidator.tokenValidators")
+					.asInstanceOf(InstanceOfAssertFactories.collection(OAuth2TokenValidator.class))
+					.hasSize(1)
+					.first()
+					.extracting("tokenValidators")
```

Comment:
These changes should be reverted. The suggestion above should allow the original test to pass.

Reply:
Reverted